.github/workflows/ci.yml

```diff
@@ -41,37 +43,14 @@ jobs:
         run: |
           source ci-scripts/validate-e2e-execution.sh
           echo "::set-output name=SHOULD_RUN_E2E::$(echo $RUN_E2E)"
-  unit_tests_core:
+  unit_tests:
```

Comment:
One of the reason the tests were split into 2 jobs/files was to reduce the run time by providing parallellism.  Does the new way of doin things make it longer to run the unit test suite when running the whole suite?

Reply:
Does the new way of doin things make it longer to run the unit test suite when running the whole suite?

yup because it will run all. To me it's a must to run all now since we're only running affected ones in pull_requests